sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,111 @@
+-- Phasing
+DELETE FROM `spell_area` WHERE `spell`=70193 AND `area`=4092;
+INSERT INTO `spell_area` (`spell`,`area`,`quest_start`,`quest_end`,`aura_spell`,`racemask`,`gender`,`autocast`,`quest_start_status`,`quest_end_status`) VALUES
+(70193,4092,24535,0,0,0,2,1,10,0),
+(70193,4092,24563,0,0,0,2,1,10,0);
```

Comment:
Does the phasing ends when the quest is rewarded? no nee to set `quest_end_status`=11?

Reply:
This spell is also used in the next quest.